.github/workflows/pr-build.yml

```diff
@@ -90,10 +103,82 @@ jobs:
           images: ghcr.io/${{ steps.normalize-repository-name.outputs.repository }}
           tags: type=ref,event=pr
 
-      - name: Build Docker image
+      - name: Build and push Docker image (internal PR)
+        if: steps.pr-check.outputs.is_external == 'false'
         uses: docker/build-push-action@v6
         with:
           context: .
           push: true
           tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels  }}
+          labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Build Docker image only (external PR)
+        if: steps.pr-check.outputs.is_external == 'true'
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          push: false
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          outputs: type=docker,dest=/tmp/image.tar
+
+      - name: Upload Docker image as artifact (external PR)
+        if: steps.pr-check.outputs.is_external == 'true'
+        uses: actions/upload-artifact@v4
+        with:
+          name: docker-image-pr-${{ github.event.pull_request.number }}
+          path: /tmp/image.tar
+          retention-days: 7
+
+      - name: External PR notice
+        if: steps.pr-check.outputs.is_external == 'true'
+        run: |
+          echo "ℹ️ This PR is from an external contributor."
+          echo "✅ Docker image was built successfully and saved as artifact."
+          echo "📦 Download the artifact 'docker-image-pr-${{ github.event.pull_request.number }}' to get the image."
+          echo "🔧 Load it with: docker load < image.tar"
+          echo "🔄 The image will be built and pushed to registry when the PR is merged."
+
+      - name: Comment on PR with artifact instructions
+        if: steps.pr-check.outputs.is_external == 'true'
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const prNumber = context.payload.pull_request.number;
+            const repository = context.repo.repo.toLowerCase();
+            const owner = context.repo.owner.toLowerCase();
+            const runId = context.runId;
+
+            const comment = `## 🐳 Docker Image Available as Artifact
+
+            Your PR Docker image has been built and is available for download!
+
+            ### 📦 Download & Load Instructions (Local Testing)
+
+            1. **Download the artifact**: Go to the [Actions tab](https://github.com/${owner}/${repository}/actions/runs/${runId}) and download \`docker-image-pr-${prNumber}\`
+            2. **Extract**: \`unzip docker-image-pr-${prNumber}.zip\`
+            3. **Load image**: \`docker load < image.tar\`
+            4. **Verify**: \`docker images | grep ${owner}/${repository}\`
```

Comment:
Is there a way to have a one-line bash and a one-line PowerShell command for the whole thing? download the image,extract,load?